### ISSUE TITLE
[Customize UI] Can I customize the User Interface of filestash using plugins or any other method. 

### ISSUE THREAD

**Issue by believerHSP** [mickael-kerjean/filestash#659]
I want to customize the User Interface of filestash. like I need to add my organization name in the landing page and also put a image in background of landing page.

Is it possible to do. If yes then what are the options available to accomplish it.

If not then, is there any opportunity that I can contribute to make this feature available in filestash?

**Comment by sevmonster** [comment]
If you are using a reverse proxy, you can inject a stylesheet into the head of the document. For example with Nginx:

```nginx
# inject custom stylesheet
sub_filter </head>
    '<link rel="stylesheet" href="/some/file.css"></head>';
sub_filter_once on;
```

NB: You must disable compression from the proxy for `sub_filter` to work.

**Comment by mickael-kerjean** [comment]
Yes, there are many ways to customise the UI that involved plugins, many of those examples can be seen in the plugin folder: https://github.com/mickael-kerjean/filestash/tree/master/server/plugin
